### ISSUE TITLE
? re GO:0060296 - regulation of cilium beat frequency involved in ciliary motility

### ISSUE THREAD

**Issue by gocentral** [geneontology/go-ontology#11831]
Hi Paola,

I was looking at this term:
GO:0060296 - regulation of cilium beat frequency involved in ciliary motility

1. I am wondering why it doesn't seem to have a direct relationship to this term:
GO:0003356 - regulation of cilium beat frequency

It seems that GO:0060296 should be is_a GO:0003356 

2. I also noticed that GO:0003356 is a direct child of this:
GO:0060295 - regulation of cilium movement involved in cell motility

and I am wondering if it should be called 
- "regulation of cilium beat frequency involved in <b>cell</b> motility"
instead of 
- "regulation of cilium beat frequency involved in ciliary motility"

thanks,

-Karen

Reported by: krchristie

Original Ticket: [geneontology/ontology-requests/11667](https://sourceforge.net/p/geneontology/ontology-requests/11667)

**Comment by paolaroncaglia** [comment]
Mentioning @krchristie as this ticket was ported over from SourceForge, so she can receive notifications.

**Comment by paolaroncaglia** [comment]
Hi Karen,

I added the link: 
GO:0060296 - regulation of cilium beat frequency involved in ciliary motility
is_a
GO:0003356 - regulation of cilium beat frequency

As for renaming ‘regulation of cilium beat frequency involved in ciliary motility’ as ‘regulation of cilium beat frequency involved in *cell* motility’, I’m unsure because, from a quick glance at the current experimental annotations to this term, it seems some of them refer to motile cilia in non-motile cells (e.g. airway epithelium). I suspect that, where annotations refer to motile cilia in motile cells, they should be to (renamed) ‘regulation of cilium beat frequency involved in *cell* motility’, while where they refer to motile cilia in non-motile cells they should simply be to ‘regulation of cilium beat frequency’. 

I think the entire node of cilium movement needs work. One underlying question is - how does a cilium move other than by beating? @JohnvanDam Then of course, a moving/beating cilium may or may not cause the entire cell to move, depending on the (unicellular) organism and on the cell/tissue type.

**Comment by JohnvanDam** [comment]
My first thought is that the term "regulation of cilium beat frequency involved in ciliary motility" contains a form of pleonasm in that cilium beating implies that the cilium is motile. Cilia/flagella movements are used for two things: 1) propelling the cell, 2) moving liquids (partially) surrounding a stationary cell. Many processes involved in ciliary beating will also be involved in both cases, with likely few exceptions. I tend to say "regulation of cilium beat frequency involved in cell motility" and have another term like "regulation of cilium beat frequency involved in fluid or mucal displacement" or somesuch. Those genes annotated for beat frequency in lung epithelial cells should then be reassigned to the new term. However I am not sure whether the distinction biologically will be relevant since I suspect beat frequency regulation in both cases will likely be caused by the same agents.

Note that in the human body only the sperm cell actually uses the flagellum for locomotion (that I know of), all others that use motile cilia are themselves stationary (e.g. fallopian tube epithelium, lung epithelial cells) and are moving liquids, mucus and other cells (the ovum). Cells that use cilia instead of flagella for locomotion are organisms like Paramecium aka "ciliates". 

Flagella tend to have a more "rotating" beat while cilia tend to have a "stroke". There are some nice gifs that show the distinction in movement quite well. From my head the movements of both types are referred to as "beating". The difference in movement however suggests that there are differences in the proteins involved in flagellar beating vs. ciliary beating. The academic question then is whether this is caused by a difference in regulation or a difference in the underlying structure and organisation of force generating components.

So in conclusion I think that the term "regulation of cilium beat frequency involved in ciliary/cell motility" could be dropped in favour of "regulation of cilium beat frequency" or we keep it in the hope that the term becomes more relevant later in time but then add the additional term. 

It all depends on the intention of the original definition of this term, i.e. "ciliary" vs. "cell" motility. In the former case it is a pleonasm and thus redundant, in the latter we need an additional term for cilia beat frequency in stationary cells.

**Comment by krchristie** [comment]
Thanks John for the great info on the difference in how sperm flagella and motile cilia on stationary multicilated epithelial cells beat. I hadn't picked up on that before.

LIke John, I am also under the impression that the sperm is the only cell in the human body that uses a flagellum for cellular motility. Sperm are a topic of a great deal of study however, so as an annotator, I have encountered papers where researchers studying sperm determined that a defect in beat frequency is a cause of a defect in sperm motility, so my perception is that this term is useful. Since the term "regulation of cilium beat frequency involved in ciliary motility"  is an is_a child of "regulation of cilium movement involved in cell motility", I think the intent of this term has been to indicate regulation of cilium beat frequency involved in cell motility. 

- GO:0003352 - regulation of cilium movement
-- is_a GO:0060295 - regulation of cilium-dependent cell motility
--- is_a GO:0060295 - regulation of cilium movement involved in cell motility
---- is_a GO:0060296 - regulation of cilium beat frequency involved in ciliary motility
-- is_a GO:0003356 - regulation of cilium beat frequency

However, with respect to John's comments about the difference in how epithelial, and other, motile cilia on stationary cells beat versus how the flagellum beat, it seems to me that we really might be dealing with two different kinds of beating on stationary cells just in mammals. John already mentioned multiciliated epithelial cells. However, there are also the motile primary cilia on the cells of the node during embryonic development. From the talks I've heard at meetings, these also have more of a rotating beat than a stroke. So, even for motile cilia on non-motile cells, it seems that there a couple different types of movement that might be worth distinguishing.

I tend to think that it might be useful to add another specific term or terms to distinguish the different processes or types of movement, however, I wonder how these terms will correlate across diverse species. So in mammals, multiciliated cells are typically epithelial cells that beat with a stroke, while sperm flagella have a rotating beat. What kind of beat do the cilia on multiciliated protozoa have? 

Would we need to make distinctions based on whether the cell has a single motile cilium (aka flagellum like in the sperm), is multiciliated, or has a motile primary cilium? 

So, I lean towards making more specific terms, but think we need to think carefully about how to do it.

-Karen

**Comment by JohnvanDam** [comment]
Hi Karen,

I forgot about the nodal cilia! You are right, they make regular circular motions.
To give you an idea of how a typical motile cilium makes a "beat", checkout this gif:
![Image of Beating Cilium](http://1.bp.blogspot.com/-S224z47YLXA/VL08YVeuJzI/AAAAAAAADWs/1uRZUfjQKcc/s1600/6-cilium.gif)

I couldn't find one for flagella unfortunately, but I have found a very enlightening youtube video on how different organisms use their flagella/cilia for locomotion (I understand that I am risking turning this post into a biology lesson):
https://www.youtube.com/watch?v=QGAm6hMysTA

We can safely assume that there are more ways of "beating" a cilium/flagellum. wiggling versus propellor comes to mind for flagella, circular versus the power stroke-recovery stroke for cilia. But for the ciliates I can imagine it might be different because they need to change direction in a moments notice, human cells rarely have to. 

To come back to the discussion at hand: I would say that cilia/flagella beating for cell motility will be more dynamically regulated to account for speed, reverse direction etc. compared to producing fluid flows for stationary cells. I suggest two distinct terms at this level, one for ciliary beating for cell motility and one for producing fluid flows (stationary cells). We could then easily attach organism specific terms to those (I am not sure if that is how it is supposed to work in GO).

Cheers,
John

**Comment by JohnvanDam** [comment]
A note to add: There are also cilia on "stationary cells" that are used for locomotion of entire multicellular organisms. For instance the planarian Schmidtea mediterranea
http://ciliajournal.biomedcentral.com/articles/10.1186/s13630-016-0037-1
But also Trichoplax adhearens. Need to figure out if locomotion of organism is caused by fluid displacement or by actually produce force along substrate. If the latter is the case this would not be covered by the two terms I suggested at the end of the previous post.

**Comment by paolaroncaglia** [comment]
Hi @krchristie  and @JohnvanDam ,

I reviewed this thread. While the biology behind how ciliated cells move is fascinating :-) I’m afraid that I’ll need to take the discussion back into the GO realm, and focus on what is within scope for GO, given our limited resources. For this reason, I think that at this time we may simply keep both terms, ‘regulation of cilium beat frequency involved in ciliary motility’ as well as its parent ‘regulation of cilium movement involved in cell motility’. The differences may be subtle depending on what wordings a paper uses. This said, if Karen or others have examples of papers that they’d want to annotate to a ‘regulation of cilium beat frequency’ term that is NOT involved in motility of the cilium, we may revisit the subject. But for now, I’ll close this ticket and focus on more pressing issues. Thanks for your patience and feedback, I learnt a lot!
